app/uk/gov/hmrc/eoricommoncomponent/frontend/services/subscription/SubscriptionService.scala

```diff
@@ -31,11 +31,11 @@ import uk.gov.hmrc.http.HeaderCarrier
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class SubscriptionService @Inject() (connector: SubscriptionServiceConnector, config: Sub02Config)(implicit
+class SubscriptionService @Inject() (connector: SubscriptionServiceConnector, config: FeatureFlags)(implicit
```

Comment:
not sure if it's worth, but you can change name of the config variable to featureFlags